### PR TITLE
chore(rxTokenInterceptor): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-502

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
DO NOT MERGE: files incorrectly named

File: src/rxTokenInterceptor/rxTokenInterceptor.module.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * @ngdoc overview
+ * @name rxTokenInterceptor
+ * @description
+ * # rxTokenInterceptor Component
+ *
+ * The rxTokenInterceptor component adds an authorization token to all http requests.  This
```
Comment (parlarjb) [inline src/rxTokenInterceptor/rxTokenInterceptor.module.js]: There's an extra space between the period and "This" (I know it's not your fault, but might as well fix now)

Comment (CITguy) [inline src/rxTokenInterceptor/rxTokenInterceptor.module.js]: fixed

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**